src/components/app/details/cdDetails/DeploymentDetailSteps.tsx

```diff
@@ -54,19 +58,38 @@ export default function DeploymentDetailSteps({ deploymentStatus, deploymentAppT
         }
     }, [])
 
-    return deploymentStatus === 'Aborted' ? (
+    const redirectToDeploymentStatus = () => {
+        const newUrl = `${URLS.APP}/${appId}/${URLS.APP_DETAILS}/${envId}/status`
+        history.push(newUrl)
+    }
+
+    return deploymentStatus === 'Aborted' || deploymentStatusDetailsBreakdownData.deploymentStatus === 'superseded' ? (
```

Comment:
Is it the expected text `superseded`

Reply:
Yes it is used when deployment failed because of a new deployment trigger